src/sandstorm/app-index/keybase.h

```diff
@@ -0,0 +1,53 @@
+// Sandstorm - Personal Cloud Sandbox
+// Copyright (c) 2020 Sandstorm Development Group, Inc. and contributors
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SANDSTORM_APPINDEX_KEYBASE_H_
+#define SANDSTORM_APPINDEX_KEYBASE_H_
+
+#include <kj/string.h>
+#include <capnp/serialize.h>
+#include <sandstorm/api-session.capnp.h>
+#include <sandstorm/app-index/keybase-api.capnp.h>
+#include <sandstorm/app-index/app-index.capnp.h>
+
+namespace sandstorm {
```

Comment:
In modern C++ (either 14 or 17, can't remember) you can do `namespace sandstorm::appindex::keybase {`.

I'm not sure if we're currently compiling with C++17 but if not we should start. Or maybe even C++20...

Reply:
Done. Seems to work with our existing build flags (`-std=c++1z`).

Reply:
1z is 17. Cool.